ui/src/pages/page/PageSection.scss

```diff
@@ -20,7 +20,7 @@
       align-items: flex-start;
       justify-content: space-between;
 
-      :first-child {
+      > :first-child {
```

Comment:
nice!

Reply:
I'm no expert in CSS but this took me a rather embarrassing long time to figure out...